supportedBackends/spark/src/main/scala/cromwell/backend/impl/spark/SparkJobExecutionActor.scala

```diff
@@ -0,0 +1,207 @@
+
+  lazy val extProcess = new SparkProcess{}
+  lazy val clusterManagerConfig = configurationDescriptor.backendConfig.getConfig("cluster-manager")
+  private val fileSystemsConfig = configurationDescriptor.backendConfig.getConfig("filesystems")
+  private val sparkMaster = configurationDescriptor.backendConfig.getString("master").toLowerCase
+  private val sparkDeployMode = configurationDescriptor.backendConfig.getString("deployMode").toLowerCase
+  override val sharedFileSystemConfig = fileSystemsConfig.getConfig("local")
+  private val workflowDescriptor = jobDescriptor.descriptor
+  private val jobPaths = new JobPaths(workflowDescriptor, configurationDescriptor.backendConfig, jobDescriptor.key)
+
+  // Files
+  private val executionDir = jobPaths.callRoot
+  private val scriptPath = jobPaths.script
+
+  private lazy val stdoutWriter = extProcess.untailedWriter(jobPaths.stdout)
+  private lazy val stderrWriter = extProcess.tailedWriter(100, jobPaths.stderr)
+
+  private lazy val clusterStdoutWriter = clusterExtProcess.untailedWriter(jobPaths.stdout)
+  private lazy val clusterStderrWriter = clusterExtProcess.tailedWriter(100, jobPaths.stderr)
+
+  private val call = jobDescriptor.key.call
+  private val callEngineFunction = SharedFileSystemExpressionFunctions(jobPaths, fileSystems)
+
+  private val lookup = jobDescriptor.inputs.apply _
+
+  private val executionResponse = Promise[BackendJobExecutionResponse]()
+
+  private val runtimeAttributes = {
+    val evaluateAttrs = call.task.runtimeAttributes.attrs mapValues (_.evaluate(lookup, callEngineFunction))
+    // Fail the call if runtime attributes can't be evaluated
+    val runtimeMap = TryUtil.sequenceMap(evaluateAttrs, "Runtime attributes evaluation").get
+    SparkRuntimeAttributes(runtimeMap, jobDescriptor.descriptor.workflowOptions)
+  }
+
+  /**
+    * Restart or resume a previously-started job.
+    */
+  override def recover: Future[BackendJobExecutionResponse] = {
+    log.warning("{} Spark backend currently doesn't support recovering jobs. Starting {} again.", tag, jobDescriptor.key.call.fullyQualifiedName)
+    taskLauncher
+    executionResponse.future
+  }
+
+  /**
+    * Execute a new job.
+    */
+  override def execute: Future[BackendJobExecutionResponse] = {
+    createExecutionFolderAndScript()
+    taskLauncher
+    executionResponse.future
+  }
+
+  private def executeTask(process: SparkProcess, stdoutWriter: UntailedWriter, stderrWriter: TailedWriter): Future[BackendJobExecutionResponse] = {
+    val submitResult = submitSparkScript(process, ProcessLogger(stdoutWriter.writeWithNewline, stderrWriter.writeWithNewline))
+    List(stdoutWriter.writer, stderrWriter.writer).foreach(_.flushAndClose())
+
+    resolveExecutionResult(submitResult, runtimeAttributes.failOnStderr)
+  }
+
+  private def submitSparkScript(sparkProcess: SparkProcess, processLogger: ProcessLogger): Try[Int] = {
+    val argv = Seq(SparkCommands.SHELL_CMD, scriptPath.toString)
```

Comment:
Give Seq a concrete type. In this case since you're effectively aking a List, make a List